status-server/src/public/js/client.js

```diff
@@ -42,15 +42,77 @@ function init() {
     }
   }
   // populate the LEDs with some initial colors
-  var colors = ['#ffff00', '#ffff00', '#00ffff', '#00ff07', '#772cb6'];
+  var colors = ['#ff00ff', '#ffff00', '#00ffff', '#00ff07', '#772cb6'];
   colors.forEach((color, idx) => {
     var node = document.getElementById("led" + idx);
     var ctx = node.getContext("2d");
     ctx.fillStyle = color;
     ctx.fillRect(0,0,300,300);
+    ctx.font = "10px Arial";
+    ctx.strokeText(idx ,node.width/2,node.height/2);
+
+
   });
 }
+function animate(ct){
+  var fetchResponse = window.fetch('http://localhost:3000/status'+ct+'.json'+Date.now()).then(
+      function onSuccess(response){
+        return response.json();
+      },
+      function onError(err){
+        console.console.warn("Error fetching"+ url, err);
+      }
+    );
+           updateValueStatus(ct);
```

Comment:
I think your intention here is to call updateValueStatus when the result from the fetch is available. fetch is asynchronous, so if you add some console.logs in here, you'll the following sequence: 
1. top of animate function
2. updateValueStatus is called
3. onSuccess/onError callback function is called with the result

See above comment, but if you do want to run code when the result is received, you'll need to provide the callbacks via the Promise returned by window.fetch, i.e. 
fetchResponse.then(function(resp) { 
  // do something in the success case
}, function(err) {
  // do something in the failure case
});